dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JSONEventsTest.java

```diff
@@ -101,7 +101,8 @@ public void checkFfdc() throws Exception {
                                                                                       "ibm_sequence", "ibm_className", "ibm_exceptionName", "ibm_probeID",
                                                                                       "ibm_threadId", "ibm_stackTrace", "ibm_objectDetails"));
 
-        ArrayList<String> ffdcKeysOptionalList = new ArrayList<String>();
+        //JDK 15 has message as an additional field in ffdc events
+        ArrayList<String> ffdcKeysOptionalList = new ArrayList<String>(Arrays.asList("message"));
```

Comment:
I am suggesting to keep `checkFfdc()` for Java 14 and lower.  Create a new method for Java 15 and make the message field mandatory.